src/support/import-supervisor.js

```diff
@@ -142,14 +145,40 @@ function ImportSupervisor(services, config) {
     }
   }
 
+  async function writeImportScriptToS3(jobId, importScript) {
+    if (!hasText(importScript)) {
+      throw new ErrorWithStatusCode('Bad Request: importScript should be a string', 400);
+    }
+
+    // Check for the length of the importScript
+    if (importScript.length > maxLengthImportScript) {
+      throw new ErrorWithStatusCode(`Bad Request: importScript should be less than ${maxLengthImportScript} characters`, 400);
+    }
+
+    let decodedScript;
+    try {
+      decodedScript = Buffer.from(importScript, 'base64').toString('utf-8');
+    } catch (error) {
+      throw new ErrorWithStatusCode('Bad Request: importScript should be a base64 encoded string', 400);
+    }
+
+    const key = `import/${jobId}/import.js`;
+    const command = new PutObjectCommand({ Bucket: s3Bucket, Key: key, Body: decodedScript });
+    try {
+      await s3Client.send(command);
+    } catch (error) {
```

Comment:
it seems you're not making use of the `error` param, as such you can use `} catch {` directly